share/python/driver.py

```diff
@@ -442,7 +442,7 @@ def energy(name, **kwargs):
     wfn = procedures['energy'][lowername](lowername, molecule=molecule, **kwargs)
 
     for postcallback in hooks['energy']['post']:
-        postcallback(lowername, **kwargs)
+        postcallback(lowername, wfn=wfn, **kwargs)
 
```

Comment:
Eventually, can take method strings from https://github.com/cdsgroup/qcdb/blob/master/qcdb/modelchems.py#L355 or the like.

I'm going to merge in https://github.com/psi4/psi4/pull/376 as soon as any one travis completes.

If any developers complain to you of failing tests, tell them to rm their *pyc files. With the driver reorg, the old pyc can confuse things (like what exception should be caught).